src/prefect/server/api/csrf_token.py

```diff
@@ -17,7 +19,16 @@ async def create_csrf_token(
     client: str = Query(..., description="The client to create a CSRF token for"),
 ) -> schemas.core.CsrfToken:
     """Create or update a CSRF token for a client"""
+    if PREFECT_SERVER_CSRF_PROTECTION_ENABLED.value() is False:
+        raise HTTPException(
+            status_code=status.HTTP_422_UNPROCESSABLE_ENTITY,
+            detail="CSRF protection is disabled.",
+        )
```

Comment:
why do we do a 422 here over say a 400?

Reply:
A 400 suggests that the user did something wrong, it's a 'bad request', which isn't super accurate here. The 422 is more so, saying that shape of the request was fine but we weren't able to process it for some reason, and in this case that reason is that it's not configured. More _technically_ this should probably be a 503, saying that the CSRF service is unavailable, but because we do auto-retries on 503s I wanted to avoid it.

Reply:
Right, that makes sense, thanks for clarifying.